core/src/main/kotlin/com/gitlab/kordlib/core/behavior/RoleBehavior.kt

```diff
@@ -37,7 +37,12 @@ interface RoleBehavior : Entity, Strategizable {
     /**
      * The raw mention of this entity.
      */
-    val mention: String get() = "<@&${id.value}>"
+    val mention: String get() {
+        return when(guildId.value == id.value) {
+            true -> "@everyone"
+            false -> "<@&${id.value}>"
+        }
```

Comment:
Prefer using `if/else` for binary conditions: https://kotlinlang.org/docs/reference/coding-conventions.html#if-versus-when

```suggestion
        return if(guildId.value == id.value) "@everyone"
        else "<@&${id.value}>"
```

Reply:
Hi, thanks you for the tip, I'm going to consider it next time :)